### ISSUE TITLE
moan is not working on sample project

### ISSUE THREAD

**Issue by airmrcr** [neocotic/moan#11]
There's something either very wrong with the sample project or with moan itself so this is a critical bug.
### Steps to reproduce
1. Install moan globally:

``` bash
$ npm install -g moan
```
1. Checkout sample project:

``` bash
$ git clone https://github.com/neocotic/moan-sample.git moan-sample
```
1. Install sample project:

``` bash
$ cd moan-sample
moan-sample $ npm install
```
1. Nothing happens when using either npm script or global moan:

``` bash
moan-sample $ npm test

> moan-sample@0.1.0 test /path/to/moan-sample
> moan

moan-sample $ moan

```

**Comment by airmrcr** [comment]
It might help to have #9 and #10 implemented first to help debug this problem as currently impossible to get extra information to track the steps and find where it's failing.

**Comment by airmrcr** [comment]
This has been fixed and will be included in the 0.1.2 release.
